### PR TITLE
chore: clean up Vision quickstart

### PR DESCRIPTION
**Pull request by telpirion**
## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: vision/snippets/quickstart/quickstart_test.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 import quickstart
 
 
-def test_quickstart() -> None:
-    labels = quickstart.run_quickstart()
-    for label in labels:
-        assert len(label.description) > 0
+def test_quickstart(capsys) -> None:
```
Comment (rsamborski) [inline vision/snippets/quickstart/quickstart_test.py:18]: Recently we've been updating multiple samples to stop using capsys and add return values to individual functions. It seems that you're doing the opposite here. I suggest to keep returning the labels and use them in tests.

Comment (telpirion) [reply vision/snippets/quickstart/quickstart_test.py:18]: Hmm, that practice doesn't align with the Samples Style guide:
https://googlecloudplatform.github.io/samples-style-guide/#result

**Review by kweinmeister — APPROVED** [review]

File: vision/snippets/quickstart/quickstart_test.py (hunk 1 above)
Comment (rsamborski) [reply vision/snippets/quickstart/quickstart_test.py:18]: I have reached out via chat to discuss this in context of [another PR](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/9323#discussion_r1260892266) with similar request.

Comment (telpirion) [reply vision/snippets/quickstart/quickstart_test.py:18]: Since a related docs change has already been submitted, I'll accept the requested changes.

**Review by telpirion — COMMENTED** [review]
Per reviewer

**Review by nicain — APPROVED** [review]

File: vision/snippets/quickstart/quickstart.py [hunk 2]
```diff
@@ -52,7 +48,4 @@ def run_quickstart() -> vision.EntityAnnotation:
     return labels
```
[the remark this answers is not in the document]
Comment (telpirion) [reply vision/snippets/quickstart/quickstart.py:50]: Done

**Pull request merged**